src/Compilers/Core/Portable/SourceGeneration/GeneratorDriverOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Options passed to a <see cref="GeneratorDriver"/> during creation
+    /// </summary>
+    public readonly struct GeneratorDriverOptions
+    {
+        public static GeneratorDriverOptions Default = new GeneratorDriverOptions(IncrementalGeneratorOutputKind.None);
```

Comment:
>Default [](http://example.com/codeflow?start=45&length=7)

Minor point: This field seems unnecessary since the type is a `struct` so it's inexpensive to create at each call site and because the value is equivalent to `default(GeneratorDriverOptions)`.